src/org/usfirst/frc/team125/robot/commands/ReleaseCarrierCMD.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc.team125.robot.commands;
+
+import org.usfirst.frc.team125.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ReleaseCarrierCMD extends Command{
+
+	public ReleaseCarrierCMD() {
+		requires(Robot.doubleLift);
+	}
+	
+	protected void initialize() {
+		Robot.doubleLift.releaseCarrier();
+	}
+	
+	protected void excecute() {
```

Comment:
Let's get rid of all the typos in command. I know this one doesn't do anything but let's be safe.